src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -598,6 +598,30 @@ private void DisableOngoingTasksIfNeeded(DatabaseRecord databaseRecord)
                     task.Disabled = true;
                 }
             }
+
+            if (databaseRecord.ExternalReplications != null)
+            {
+                foreach (var task in databaseRecord.ExternalReplications)
+                {
+                    task.Disabled = true;
```

Comment:
Shouldn't this come from the restore config?

Reply:
We skip it if it was enabled in the restore config:
https://github.com/ravendb/ravendb/blob/462b73c58d4d9d7f48081dc305e053b9cbed3efa/src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs#L567